data/campaigns/The_South_Guard/scenarios/03_A_Desparate_Errand.cfg

```diff
@@ -550,7 +550,7 @@
             name=EthilielV.modifications
             mode=insert
             [value]
-                {TRAIT_LOYAL}
+                {TRAIT_LOYAL OVERLAY=""}
```

Comment:
I can't justify it, but my gut feeling is that TRAIT_LOYAL_NO_OVERLAY feels better than a macro with an argument.

Reply:
we could, i don't have a strong opinion on it.